app/services/ucas_matching/upload_matching_data.rb

```diff
@@ -0,0 +1,35 @@
+require 'csv'
+
+module UCASMatching
+  class UploadMatchingData
+    include Sidekiq::Worker
+
+    def perform
+      filename = "/tmp/dfe_apply_itt_applications_#{DateTime.now}.csv"
+
+      File.open(filename, 'a') do |f|
```

Comment:
If this file is unique by timestamp, would `w` mode make more sense?

Reply:
Done!